blocks/results-list-block/features/results-list/results/result-item.jsx

```diff
@@ -28,59 +49,100 @@ const ResultItem = React.memo(
 				display_date: displayDate,
 				headlines: { basic: headlineText } = {},
 				websites,
+				credits,
 			} = element;
+			const {
+				dateLocalization: { language, timeZone, dateTimeFormat } = {
+					language: "en",
+					timeZone: "GMT",
+					dateTimeFormat: "LLLL d, yyyy 'at' K:m bbbb z",
+				},
+			} = getProperties(arcSite);
+			const phrases = getTranslatedPhrases(getProperties(arcSite).locale || "en");
+			const { registerSuccessEvent } = useComponentContext();
 
-			const imageURL = extractImageFromStory(element);
+			/* Author Formatting */
+			const imageURL = getImageFromANS(element)?.url || null;
+			const { searchableField } = useEditableContent();
+			const hasAuthors = showByline ? credits?.by && credits?.by.length : null;
+			const contentHeading = showHeadline ? headlineText : null;
+			const formattedDate = Date.parse(displayDate)
+				? localizeDateTime(new Date(displayDate), dateTimeFormat, language, timeZone)
+				: "";
 			const url = websites[arcSite].website_url;
-			return (
-				<div className={`list-item ${!showImage ? "no-image" : ""}`} ref={ref}>
+			return showHeadline ||
+				showImage ||
+				showDescription ||
+				showByline ||
+				showDate ||
+				showItemOverline ? (
+				<div
+					ref={ref}
+					className={`${BLOCK_CLASS_NAME}${showImage ? ` ${BLOCK_CLASS_NAME}--show-image` : ""}`}
+				>
 					{showImage ? (
-						<div className="results-list--image-container">
-							<a href={url} title={headlineText} aria-hidden="true" tabIndex="-1">
+						<MediaItem {...searchableField("imageOverrideURL")} suppressContentEditableWarning>
+							<Conditional
+								component={Link}
+								condition={url}
+								href={formatURL(url)}
+								onClick={registerSuccessEvent}
+								assistiveHidden
+							>
 								<Image
```

Comment:
We need to add the `auth` token to use resizer v2

Reply:
resizedOptions was added (line 97)